Sources/PointFree/Auth.swift

```diff
@@ -1,15 +1,19 @@
+import Css
 import Either
 import Foundation
 import Html
+import HtmlCssSupport
 import HttpPipeline
 import HttpPipelineHtmlSupport
 import Optics
 import Prelude
+import Styleguide
 import UrlFormEncoding
 
 let secretHomeResponse: (Conn<StatusLineOpen, Prelude.Unit>) -> IO<Conn<ResponseEnded, Data>> =
   writeStatus(.ok)
     >-> readGitHubSessionCookieMiddleware
+    >-> (ignoreErrors >>> pure)
```

Comment:
just some middleware to convert `Either<E, A>` to `A?`. there are a few other ways to write this, so down to experiment.